packages/cli/src/lib/strings.ts

```diff
@@ -248,4 +253,17 @@ the PWA:
   warnPwaFailedQuality: red('PWA Quality Criteria check failed.'),
   updateConfigUsage: 'Usage\n\n:[--jdk <path-to-jdk>] [--androidSdk <path-to-android-sdk>]' +
       '(You can insert one or both of them)',
+  jdkPathIsNotValid: 'The jdkPath doesn\'t exist, please run the following command to update it:' +
+      '\nbubblewrap updateConfig --jdkPath <path-to-jdk>. Then run bubblewrap doctor again.',
+  jdkPathIsNotCorrect: 'The jdkPath isn\'t correct, please run the following command to update ' +
```

Comment:
I'm not quite sure why we need these to be separate - they're both essentially the same thing right?

Reply:
Your'e right, it's redundant.. I will change it